src/utils/externalLinks.test.js

```diff
@@ -5,7 +5,7 @@ import history from '../history';
 import routes from '../constants/routes';
 
 describe('externalLinks', () => {
-  const historyPush = spy(history, 'push');
+  const historyReplace = spy(history, 'replace');
```

Comment:
You should use `mock` instead of `spy` to prevent `HeadlessChrome 65.0.3325 (Linux 0.0.0) ERROR Some of your tests did a full page reload!` in https://jenkins.lisk.io/job/lisk-hub/job/PR-677/1/console